app/services/foods_service.rb

```diff
@@ -0,0 +1,22 @@
+class FoodsService
+
+  def self.conn
+      Faraday.new(
+      url: "https://api.nal.usda.gov/fdc/",
+      params: {},
+      headers: {"X-Api-Key" => "#{ENV["FOOD_DATA_API_KEY"]}"
+    }
+    )
+  end
+
+  def self.search_food(query)
+    response = conn.get("v1/foods/search?query=#{query}")
```

Comment:
Would have loved to see the `query` param set in a block or as an argument similar to `conn` above.